.yamato/package-test.yml

```diff
@@ -15,8 +18,15 @@ test_{{ platform.name }}_{{ editor.version }}:
     UPMCI_ENABLE_APV_CLEAN_CONSOLE_TEST: 1
 {% endif %}
   commands:
-     - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
-     - upm-ci package test --unity-version {{ editor.version }} --package-path com.unity.formats.alembic
+    - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+# 2019.4 and 2020.3 doesn't have Silicon version Editor, so download x64 version for them even on Silicon Mac. On other platforms, always download x64 version.
+{% if platform.model != "M1" or editor.version == "2019.4" or editor.version == "2020.3" %}
```

Comment:
I am really struggling understanding this if and how it correspond to the comment... 👀

Reply:
It means if `the platform is non-Silicon Mac, e.g. linux, win or regular Mac` or `editor.version == 2019.4` or `editor.version == 2020.3`, just run `upm-ci test` which will download X64 version of Unity editor. Because there is no Silicon Editor for 2019.4 and 2020.3. 
Basically this condition equals 
`{% if platform.model != "M1" or platform.model == "M1" and (editor.version == "2019.4" or editor.version == "2020.3")%}`.   BUT liquid template language has very limited support on control flow such as 
- Parentheses are invalid characters in Liquid
- `and` or `or` operators are checked in order from right to left which is opposite to other programming languages.
Because of this, I used the short version which you are seeing.

In all other cases which means `platform.model == "M1" and editor.version != "2019.4" and editor.version != "2020.3"`, Silicon version Editor will be downloaded for testing on Silicon Mac.